### ISSUE TITLE
Odd Cursor Glitch When Moving Through Windows and Game

### ISSUE THREAD

**Issue by CheeseSprinkles** [OpenTTD/OpenTTD#8076]
## Version of OpenTTD
1.10.0, Linux Debian
## Expected result
The cursor should behave normally in-game.
## Actual result
The cursor bugs out in-game, lags significantly and creates clones of itself while moving around.
## Steps to reproduce
Open any save or start a new game; move the cursor aorund.
![Screenshot 2020-04-10 at 1 07 54 PM - Edited](https://user-images.githubusercontent.com/52007125/79020188-040e1d00-7b2d-11ea-9b38-6f12bfe59029.png)

**Comment by nielsmh** [comment]
Which video driver? SDL 1, SDL 2, or (unlikely) Allegro?

**Comment by CheeseSprinkles** [comment]
SDL1 (I think)

**Comment by CheeseSprinkles** [comment]
> Which video driver? SDL 1, SDL 2, or (unlikely) Allegro?

SDL1, I think.

**Comment by CheeseSprinkles** [comment]
Playing in 1.10.1 now; problem still persists. Also, a savegame with the issue (which i forgot to attach originally)
[test, Jan 1st, 2010.zip](https://github.com/OpenTTD/OpenTTD/files/4511359/test.Jan.1st.2010.zip)

**Comment by James103** [comment]
On Windows 10 Home 10.0.18363 Build 18363, moving the cursor a lot can cause the cursor to disappear (and/or be cut off) for a bit. Unfortunately, I was unable to capture that in a screenshot, but it happens in practice. It is better reproduced with an empty map.
[openttd.cfg.zip](https://github.com/OpenTTD/OpenTTD/files/4740639/openttd.cfg.zip)


```
Configuration:
 Blitter:      32bpp-optimized
 Graphics set: OpenGFX (7393)
 Language:     C:\Program Files\OpenTTD\lang\english.lng
 Music driver: dmusic
 Music set:    Barry Taylor's Traditional American Tunebook Volume 2 (1)
 Network:      no
 Sound driver: xaudio2
 Sound set:    OpenSFX (87)
 Video driver: win32
```

**Comment by TrueBrain** [comment]
@James103 : upcoming nightly (that of 20210223) will have that fixed (see #8726).

@CheeseSprinkles : we did a lot of improvements on rendering in general, but I have never seen this issue in all the testing I did. Now there is a chance this is fixed in our latest nightly (https://www.openttd.org/downloads/openttd-nightlies/latest.html ). Would you mind giving that a spin to see if the problem is resolved?

Tnx!

**Comment by LordAro** [comment]
Given the video drivers have essentially been rewritten in the last few months, I will consider this resolved,  as any similar will be a completely separate
